test/WebJobs.Script.Tests/TestScripts/Node/functions.tests.js

```diff
@@ -99,9 +128,9 @@ describe('http', () => {
         });
 
         it('get', () => {
-            expect(req.get('test')).to.equal('val');
```

Comment:
Yeah, I want a test that explicitly verifies that a header can be retrieved with both upper and lower case names. Easy to do in unit tests.